src/core.js

```diff
@@ -140,4 +140,11 @@ if ( jQueryVersionSince( "3.3.0" ) ) {
 		},
 		"jQuery.isWindow() is deprecated"
 	);
+
+	migrateWarnFunc( jQuery, "isArray",
+		function( a ) {
```

Comment:
would it be possible to just provide `Array.isArray` here, without a wrapper?

Reply:
Seems like it! I pushed a commit for that.